### PR TITLE
Expose forward and set it to false in default

### PR DESCRIPTION
**Pull request by shiyuhang0**
### What problem does this PR solve? 

https://github.com/tikv/client-java/pull/198

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation
 - Need to update the `tidb-ansible` repository
 - Need to be included in the release note

### CODE REVIEW

File: tikv-client/src/main/java/com/pingcap/tikv/operation/iterator/DAGIterator.java [hunk 1]
```diff
@@ -223,6 +223,7 @@ private SelectResponse process(RegionTask regionTask) {
                 .getRegionStoreClientBuilder()
                 .build(region, store, storeType);
         client.addResolvedLocks(startTs, resolvedLocks);
+        //client.setTimeout(clientSession.getConf().getTimeout(),);
```
Comment (Daemonxiao) [inline tikv-client/src/main/java/com/pingcap/tikv/operation/iterator/DAGIterator.java:226]: it seems no use here.

Comment (shiyuhang0) [reply tikv-client/src/main/java/com/pingcap/tikv/operation/iterator/DAGIterator.java]: forget to delete

**Review by zhangyangyu — APPROVED** [review]

**Review by xuanyu66 — APPROVED** [review]

**Pull request merged**